BookClub/tests/views/test_booklists_view.py

```diff
@@ -0,0 +1,101 @@
+"""Tests of the booklists_list view."""
+from django.test import TestCase, tag
+from django.urls import reverse
+from BookClub.models import User, Club, ClubMembership, Book, BookList
+
+from BookClub.tests.helpers import reverse_with_next
+
+@tag('booklist', 'book', 'user')
+@tag('booklist_view')
+class UserBooklistsViewTestCase(TestCase):
+    """Tests of the booklists_list view."""
+
+    fixtures = [
+        'BookClub/tests/fixtures/default_users.json',
+        'BookClub/tests/fixtures/books.json',
+        'BookClub/tests/fixtures/booklists.json'
+    ]
+
+    def setUp(self):
+        self.user = User.objects.get(username="johndoe")
+        self.url = reverse('booklists_list', kwargs={'username': self.user.username})
+
+    def _get_url_for_user(self, username):
+        return reverse('booklists_list', kwargs={'username': username})
+
+    def test_url(self):
+        self.assertEqual(self.url,f"/user/{self.user.username}/lists")
+
+    # def test_does_not_redirect_when_not_logged_in(self):
```

Comment:
commented out?

Reply:
commented out because the view currently does not require the user to be logged in. If we decide to change it so it does, we can uncomment this test.